WORKSPACE

```diff
@@ -0,0 +1,54 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# To update TensorFlow to a new revision,
+# a) update URL and strip_prefix to the new git commit hash
+# b) get the sha256 hash of the commit by running:
+#    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
+#    and update the sha256 with the result.
+http_archive(
+    name = "org_tensorflow",
+    patch_args = ["-p1"],
+    patch_tool = "patch",
+    patches = [
```

Comment:
We don't need to apply patches manually anymore, right? Can you remove references to `apply_patches.sh`? That should also fix the CI script:

https://github.com/pytorch/xla/blob/ce48e42abf486afc82940f24db762e15712e3459/.circleci/common.sh#L25-L28

Reply:
Actually we might rely on `apply_patches.sh` to update to the torch pin in PRs, so it's better to add a way to skip the TF patches than remove this script right now.